psl/psl/tests/attributes/relations/relations_positive.rs

```diff
@@ -249,222 +248,6 @@ fn allow_explicit_fk_name_definition() {
         .assert_relation_fk_name(Some("CustomFKName".to_string()));
 }
 
-#[test]
-fn allow_implicit_fk_name_definition() {
```

Comment:
The DML has a wiped constraint name for default values.

We test these in the ME.